src/zkbob/utils/PriorityQueue.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.13;
+
+/**
+ * @dev The structure that stores all information about the pending commitment.
+ * @param commitment commitment value to be added in the Merkle Tree.
+ * @param prover address of the prover that submitted the commitment.
+ * @param fee fee reserved for the prover who will submit the tree update proof.
+ * @param timestamp commitment timestamp.
+ */
+struct PendingCommitment {
+    uint256 commitment;
+    address prover;
+    uint64 fee;
+    uint64 timestamp;
+}
+
+/// @author Matter Labs
```

Comment:
Should we put a link or some specific license?

Reply:
I've added the copyright notice in https://github.com/zkBob/zkbob-contracts/pull/97/commits/6d16f95b0ac5a7bbefdf566e022423d8e7062308. I guess this should be sufficient, considering that Matter Labs uses the MIT License.